### ISSUE TITLE
Homebridge does not start since 0.1.3

### ISSUE THREAD

**Issue by matnoublanche** [fjhorrillo/homebridge-airzone-cloud#11]
**Describe The Bug:**
Hi there, 

Since 0.1.3 I have this error below.

Do you know why ?

**Logs:**

```
TypeError: Cannot read property 'error' of undefined
at evaluate (/usr/lib/node_modules/homebridge-airzone-cloud/src/interface/config.ts:23:18)
at Object.isValid (/usr/lib/node_modules/homebridge-airzone-cloud/src/interface/config.ts:38:24)
at new AirzoneCloudHomebridgePlatform (/usr/lib/node_modules/homebridge-airzone-cloud/src/platform.ts:50:36)
at /var/www/html/plugins/homebridge/resources/node_modules/homebridge/src/server.ts:440:40
at Array.forEach (<anonymous>)
at Server.loadPlatforms (/var/www/html/plugins/homebridge/resources/node_modules/homebridge/src/server.ts:372:27)
at Server.start (/var/www/html/plugins/homebridge/resources/node_modules/homebridge/src/server.ts:159:29)
```

**Environment:**

* **Plugin Version**: 0.1.3
* **Homebridge Version**: 1.3.4
* **Node.js Version**: v14.17.4
* **NPM Version**: v6.14.14
* **Operating System**: Linux Jeedom 4.9.0-9-amd64

**Comment by fjhorrillo** [comment]
Hello, this is because the airzone api was changed. I’m working to implement the new api

**Comment by matnoublanche** [comment]
Hi @fjhorrillo,

No, when I downgrade to 0.1.2 it works again.

I'm still using the old app which the old API.

Since 0.1.3 you changed the log way and it seems like Homebridge does not like it at starting.

**Comment by fjhorrillo** [comment]
Hello @matnoublanche, maybe it is related to the api upgrade, the 0.1.2 is still working?

**Comment by linkweaver** [comment]
Hi @fjhorrillo,

In fact the problem was on the NPM logger module.

It was not installed by default and it's not on the dependencies of the package.json file.

I think I can install it manually but I think that it's better to declare it on the dependencies of the module.

Do you know if other encounters the same problem as me ?

**Comment by matnoublanche** [comment]
Hi @fjhorrillo,

In fact the problem was on the NPM logger module.

It was not installed by default and it's not on the dependencies of the package.json file.

I think I can install it manually but I think that it's better to declare it on the dependencies of the module.

Do you know if other encounters the same problem as me ?

**Comment by fjhorrillo** [comment]
Hi, with version 0.1.3 I had this problem but in the end I went back to using the standard log by adding a new class to print debug logs as information if the plugin debug option is on.  This was added in version 0.1.4.
What dependency do you want to add?

**Comment by linkweaver** [comment]
Ok I think that you still using Logger on the 0.1.4 version, so no need to install this as dependency.

I think I found the origin of the error. 

On the src/interface/config.ts file, lines 23 and 27 you have to prefix your lines with _this_

this.platform.log.error

**Comment by matnoublanche** [comment]
Ok I think that you still using Logger on the 0.1.4 version, so no need to install this as dependency.

I think I found the origin of the error.

On the src/interface/config.ts file, lines 23 and 27 you have to prefix your lines with _this_

`this.platform.log.error`

**Comment by fjhorrillo** [comment]
Hi, I found the problem, it was due to the log isn`t initialized yet. Fixed in version 0.1.5
